nautobot/dcim/tests/test_views.py

```diff
@@ -562,6 +563,30 @@ def test_powerports(self):
         self.assertContains(response, total_utilization_html, html=True)
 
 
+class HardwareFamilyTestCase(ViewTestCases.OrganizationalObjectViewTestCase):
+    model = HardwareFamily
+
+    @classmethod
+    def setUpTestData(cls):
+        # FIXME(jathan): This has to be replaced with# `get_deletable_object` and
```

Comment:
Probably easier to just implement the get_deletable_object? If not, remove Jathan's name.